Sources/UI/Extensions/UIKit/UIEdgeInsets+Extensions.swift

```diff
@@ -8,7 +8,7 @@
 
 import UIKit
 
-extension UIEdgeInsets {
+extension UIEdgeInsets: Hashable {
```

Comment:
Why do we need this?

Reply:
To avoid a big custom `Hashable` implementation in other places.